data/data/rhcos.json

```diff
@@ -1,135 +1,137 @@
 {
-    "amis": {
-        "ap-northeast-1": {
-            "hvm": "ami-02bb6c599c86663bd"
-        },
-        "ap-northeast-2": {
-            "hvm": "ami-03dd9e368f20dbc76"
-        },
-        "ap-south-1": {
-            "hvm": "ami-0192efd438962e43a"
-        },
-        "ap-southeast-1": {
-            "hvm": "ami-0fc688622e368c765"
-        },
-        "ap-southeast-2": {
-            "hvm": "ami-083ae473d8d6b7771"
-        },
-        "ca-central-1": {
-            "hvm": "ami-0e3eba4d065f52d54"
-        },
-        "eu-central-1": {
-            "hvm": "ami-03c14a5bb0eadf4e3"
-        },
-        "eu-north-1": {
-            "hvm": "ami-027eb49da627f8899"
-        },
-        "eu-west-1": {
-            "hvm": "ami-0fda05c1fd2f78f3d"
-        },
-        "eu-west-2": {
-            "hvm": "ami-0fd3f4561265de484"
-        },
-        "eu-west-3": {
-            "hvm": "ami-05ee3993a0d19102e"
-        },
-        "me-south-1": {
-            "hvm": "ami-013932b8aad1e340f"
-        },
-        "sa-east-1": {
-            "hvm": "ami-000abe9467b83a7db"
-        },
-        "us-east-1": {
-            "hvm": "ami-07b760584db62cbb2"
-        },
-        "us-east-2": {
-            "hvm": "ami-005141ec3f197b7be"
-        },
-        "us-west-1": {
-            "hvm": "ami-05bd75163809cc646"
-        },
-        "us-west-2": {
-            "hvm": "ami-0a2ce8ff0118b47a7"
-        }
-    },
-    "azure": {
-        "image": "rhcos-44.81.202001030903.0-azure.x86_64.vhd",
-        "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-44.81.202001030903.0-azure.x86_64.vhd"
-    },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/44.81.202001030903.0/x86_64/",
-    "buildid": "44.81.202001030903.0",
-    "gcp": {
-        "image": "rhcos-44-81-202001030903-0",
-        "url": "https://storage.googleapis.com/rhcos/rhcos/44.81.202001030903.0.tar.gz"
-    },
-    "images": {
-        "aws": {
-            "path": "rhcos-44.81.202001030903.0-aws.x86_64.vmdk.gz",
-            "sha256": "8990547b4054331c97dc60a6049be028a77e1a4e2863b3bfa6e6c07247a7d536",
-            "size": 880505842,
-            "uncompressed-sha256": "b92479b7e7c463d9f6691f3c48142e7d6bb10618b744247610437ce2ca59447c",
-            "uncompressed-size": 898413056
+    "amd64": {
```

Comment:
Should we use x86_64 for consistency with our distros/rpms throughout this PR?

Reply:
I agree that `x86_64` would probably a better key here -- though it's not a huge deal IMHO.

Reply:
I opted for `amd64` here because this value is plumbed through from the install config. Since OpenShift users are already used to the Golang names, I thought it made sense to stick with that scheme. @dgoodwin is either option easier for Hive than the other?

Reply:
Neither is a problem on the Hive side it just looked a little off in a RH world to me. Not a huge deal either way.